lib/screens/map_view.dart

```diff
@@ -85,18 +88,43 @@ class _MapViewScreenState extends State<MapViewScreen> {
           MarkerLayerOptions(
             markers: [
               Marker(
-                  width: 40.0,
-                  height: 40.0,
+                  width: 110.0,
+                  height: 110.0,
                   point: latlng.LatLng(locationData?.latitude ?? 51.5,
                       locationData?.longitude ?? -0.09),
-                  builder: (ctx) => const Beastie())
+                  builder: (ctx) => const Avatar()),
+              spawnBeastie(),
+              spawnBeastie(),
+              spawnBeastie()
             ],
           ),
         ],
       );
     }
   }
 
+  Marker spawnBeastie() {
+    var random = Random();
+    double longitudeRange = 0.00085;
+    double latitudeRange = 0.00088;
+    int randNum = random.nextInt(2);
+    int sign = 0;
+    if (randNum == 0) {
+      sign = -1;
+    } else {
+      sign = 1;
+    }
+    double randomLon = sign * (random.nextDouble() * longitudeRange + .00015);
+    double randomLat = sign * (random.nextDouble() * latitudeRange + .00032);
+    debugPrint('($randomLat, $randomLon)');
+    return Marker(
+        width: 40.0,
+        height: 40.0,
+        point: latlng.LatLng((locationData?.latitude ?? 51.5) + randomLat,
```

Comment:
I think we should be at a point now where we should remove all of the backup values (i.e. we shouldn't need `?? 51.5` for all of the point values of these markers and stuff). I might be wrong though. I guess I figure if the location isn't working, the app should probably just give an error message instead of placing the user in some random place.

Reply:
But up to you if that's a thing you want to address in this pr or we can make a ticket for that and do it as a separate item

Reply:
I can do it here, it should be very quick to fix. You're right that they definitely should be removed by now since I already made changes that guarantee they will never be used. That code will never be reached if location data is not found.